tests/Feature/Console/MergeUsersTest.php

```diff
@@ -0,0 +1,143 @@
+<?php
+
+namespace Tests\Feature\Users\Console;
+
+use App\Models\Accessory;
+use App\Models\Asset;
+use App\Models\Consumable;
+use App\Models\LicenseSeat;
+use App\Models\User;
+use App\Models\Actionlog;
+use Tests\TestCase;
+
+
+class MergeUsersTest extends TestCase
+{
+    public function testAssetsAreTransferredOnUserMerge()
+    {
+        $user1 = User::factory()->create(['username' => 'user1']);
+        $user_to_merge_into = User::factory()->create(['username' => 'user1@example.com']);
+
+        Asset::factory()->count(3)->assignedToUser($user1)->create();
+        Asset::factory()->count(3)->assignedToUser($user_to_merge_into)->create();
+
+        $this->artisan('snipeit:merge-users')->assertExitCode(0);
+
+        $this->assertNotEquals(3, $user_to_merge_into->refresh()->assets->count());
```

Comment:
Nitpick: this line can go away because the line below covers it.

Reply:
I preferred being explicit there too, but again, not a hill.